src/components/Calculator/Buttons.jsx

```diff
@@ -0,0 +1,51 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly consider avoiding disabling linters can identify common coding mistakes, potential bugs, and errors early in the development process. They analyze the code for syntax errors, unused variables, improper variable assignments, and other issues that might cause problems during runtime. Detecting and fixing these errors early saves time and effort in the debugging phase.